src/dev.c

```diff
@@ -44,6 +56,7 @@ int init_devices(void)
 		if(!dev_path_in_use(cfg.serial_dev)) {
 			dev = add_device();
 			strcpy(dev->path, cfg.serial_dev);
+			dev->evt_num = device_evt_num(cfg.serial_dev);
```

Comment:
A serial device will certainly not have a /dev/input/eventN number.

Reply:
Would there be another identifier, such as `/dev/ttyX`?

Reply:
Yes, typically it would be something like `/dev/ttySX` or `/dev/ttyUSBX`.